### PR TITLE
Get sensu-plugin-python into a useful state

### PR DESCRIPTION
**Pull request by robbydyer**
It seems this project has been abandoned. I've done some work to get it into a somewhat useful state. Most of the functionality has been mirrored from the Ruby version of sensu-plugin.

### CODE REVIEW

**Comment by robbydyer** [comment]
Ok, I've done as much style fixing as I can. Looks like the coverage tests are what is failing at this point.

File: setup.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='sensu_plugin',
-    version='0.2.0',
+    version='0.2.1',
```
Comment (luisdavim) [inline setup.py]: don't bump the version, leave that to the maintainers.

**Comment by luisdavim** [comment]
Hi @robbydyer thanks for your contribution, I know this was a long time ago but this code was left unmaintained for a while and we are now trying to get it back on track. it would be great if you're still interested in helping and could rebase your code so that we can merge it.

Thanks.

**Comment by barryorourke** [comment]
handler support has now been added into 0.4.0

**Pull request closed without merging**